packages/openapi-typescript/bin/cli.js

```diff
@@ -92,7 +95,7 @@ async function generateSchema(schema, { redoc, silent = false }) {
       exportType: flags.exportType,
       immutable: flags.immutable,
       pathParamsAsTypes: flags.pathParamsAsTypes,
-      redoc,
+      redocly,
```

Comment:
Turns out you have to name variables the same in order for them to get passed properly! Whodathunk